src/selector/transform.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#ifndef TRANSFORM_HPP
+#define TRANSFORM_HPP
+
+class Transform
```

Comment:
Why do we need Transform class? SFML already supports Transform https://www.sfml-dev.org/documentation/2.5.1/classsf_1_1Transform.php